packages/rocketchat-ui-admin/client/users/adminUsers.js

```diff
@@ -138,11 +138,11 @@ Template.adminUsers.events({
 			e.preventDefault();
 		}
 	},
-	'keyup #users-filter'(e, t) {
+	'keyup #users-filter': _.debounce((e, t) => {
 		e.stopPropagation();
 		e.preventDefault();
 		t.filter.set(e.currentTarget.value);
-	},
+	}, 300),
```

Comment:
Could this number be a constant defined somewhere? Better yet if it depends on rate limiter parameters.